src/pybind/mgr/dashboard/frontend/src/app/shared/api/osd.service.ts

```diff
@@ -76,6 +76,12 @@ export class OsdService {
     return this.http.get(`${this.path}`);
   }
 
+  getOsdSettings() {
```

Comment:
If we don't properly type all the front-end and back-end pieces, we don't really leverage the safety of type checks:
```suggestion
  getOsdSettings() : Observable<OSDSettings> {
```

Reply:
Done